website/games.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+require_once __DIR__ . '/vendor/autoload.php';
+
+use Carbon\Carbon;
+use Cocur\HumanDate\HumanDate;
+
+$config = require '/var/creds/creds.php';
```

Comment:
would be neat to update this to use the same server.config file potentially that I just added in #82. In a follow up ofc

Reply:
I really just wanted to comment and close this one. Not sure how to do that. Feel free to resolve :)